package.json

```diff
@@ -6,33 +6,47 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "deploy:s3": "serverless client deploy --no-config-change --no-policy-change --no-cors-change",
+    "build:deploy": " npm run build && npm run deploy:s3",
+    "client:deploy": "sls client deploy --no-config-change --no-policy-change --no-cors-change",
```

Comment:
is it a duplicate command of the one on line 10?

Reply:
> Could you please add links to S3 and CloudFront to README.md as it [described in task](https://github.com/rolling-scopes-school/nodejs-aws-tasks/blob/main/task2-serve-spa-aws/task.md)? I think it might be useful in future tasks.

Added, thanks for the suggestion.

Reply:
It was : )